core/modules/ccontrol/UserQueryDrop.cc

```diff
@@ -41,12 +41,7 @@
 #include "util/IterableFormatter.h"
 
 namespace {
-
-LOG_LOGGER getLogger() {
-    static LOG_LOGGER logger = LOG_GET("lsst.qserv.ccontrol.UserQueryDrop");
-    return logger;
-}
-
+    LOG_LOGGER _log = LOG_GET("lsst.qserv.ccontrol.UserQueryDrop");
```

Comment:
I found it easier to read in cases we have just a line or two, but I am happy to switch back to make it consistent with everything else.